src/components/MoNavbar.vue

```diff
@@ -70,7 +71,7 @@
         this.$store.dispatch(AUTH_LOGOUT, vm.user)
         .then(response => {
           vm.isLoading = false
-          vm.$router.push({name: 'Login'})
+          window.location.replace('/')
```

Comment:
Perhaps we should do the login prompt in a popup? That's vaguely nicer, and quite customary.

Reply:
I'm not exactly sure what you mean. Have Flask generate a login popup?

Reply:
> I'm not exactly sure what you mean. Have Flask generate a login popup?

The usual workflow is that the application triggers a popup at the remote site, you do the login, and redirect back to a page in the application. Due to having the same origin, that page can access its opener, and inform it that a successful login occurred. The main justification is that this allows you to retain application state during the login.